app/assets/javascripts/application.js

```diff
@@ -91,7 +91,7 @@ $(document).ready(function() {
   });
 
   var hash = location.hash
-    , hashPieces = hash.split('?')
+    , hashPieces = ((hash.split('?')[0] == "") ? 0 : hash.split('?'))
```

Comment:
Comma warnings can be turned off with 'laxcomma'.<br>Mixed double and single quotes.<br>Use '===' to compare with ''.<br>Bad line breaking before ','.